test/dates/accessors.jl

```diff
@@ -118,7 +109,20 @@ function test_dates2(from,to)
         end
     end
 end
-test_dates2(0,2100)
+
+# test hour, minute, second
+let h=0, mi=0, s=0, ms=0, us=0, ns=0
+    for h = 0:23, mi = 0:59, s = 0:59,
+        ms in (0,1,500,999), us in (0,1,500,999), ns in (0,1,500,999)
```

Comment:
Thank you for the info.  It will be v,good solution --the 64 bit GC wants the same adjustment.

Knowing that nothing in this PR gives rise to the GC failure by its manner of work, rather the numerosity of that test has brought to our awareness that *deeper bug*.  Do we feel it helpful to allow this (with the  lesserly agglomerative test replacing the [new accessor test](https://github.com/JuliaLang/julia/pull/12274/files#diff-2f3dfed48f169e03378a18a8e7db28d4R114)) to merge.  I think it important that people have some time with this *Time* before v0.6 rolls into home and hearth.     

    

Any few brief wants of the dev community would make it all the nicer for the rest of us>